src/main/cli/cli.c

```diff
@@ -6664,8 +6659,10 @@ static void cliHelp(const char *cmdName, char *cmdline)
 static void processCharacter(const char c)
 {
     if (bufferIndex && (c == '\n' || c == '\r')) {
-        // enter pressed
-        cliPrintLinefeed();
+        if (cliInteractive) {
+            // enter pressed, and in cli mode - so output a newline
```

Comment:
```suggestion
            // echo newline back
```
Or maybe use cliEcho(char c); that will handle interactive testing and newline

Reply:
outside of processChar no needed use of cliEcho(c) found, so leaving as if statements for now.